### ISSUE TITLE
pidgin-sipe infinite loop when copying uri from chat window

### ISSUE THREAD

**Issue by gy-lehel** [tieto/sipe#23]
gdb) where
#0  0x00007fad4b979a71 in g_object_ref () from /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0
#1  0x00007fad4e2800f4 in ?? () from /usr/lib/x86_64-linux-gnu/libwebkitgtk-3.0.so.0
#2  0x00007fad4e2825bd in webkit_dom_node_get_parent_node () from /usr/lib/x86_64-linux-gnu/libwebkitgtk-3.0.so.0
#3  0x00007fad50acceff in do_popup_menu (webview=webview@entry=0x556d963e1290, button=3, time=424808907, context=6, node=0x556d961fa360,

```
uri=0x556d96208400 "https://www.facebook.com/ProgrammersCreateLife/photos/a.241809332534619.55240.241806149201604/980463552002523/?type=3&theater") at gtkwebview.c:768
```
#4  0x00007fad50acdca5 in webview_button_pressed (webview=0x556d963e1290, event=0x556d97113ba0) at gtkwebview.c:988
#5  0x00007fad4c999c4c in ?? () from /usr/lib/x86_64-linux-gnu/libgtk-3.so.0
#6  0x00007fad4b9753b8 in g_closure_invoke () from /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0
#7  0x00007fad4b986d3d in ?? () from /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0
#8  0x00007fad4b98e6f9 in g_signal_emit_valist () from /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0
#9  0x00007fad4b98ece2 in g_signal_emit () from /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0
#10 0x00007fad4cac2974 in ?? () from /usr/lib/x86_64-linux-gnu/libgtk-3.so.0
#11 0x00007fad4c99808c in ?? () from /usr/lib/x86_64-linux-gnu/libgtk-3.so.0
#12 0x00007fad4c99991a in gtk_main_do_event () from /usr/lib/x86_64-linux-gnu/libgtk-3.so.0
#13 0x00007fad4c59dc22 in ?? () from /usr/lib/x86_64-linux-gnu/libgdk-3.so.0
#14 0x00007fad50751e04 in g_main_context_dispatch () from /lib/x86_64-linux-gnu/libglib-2.0.so.0
#15 0x00007fad50752048 in ?? () from /lib/x86_64-linux-gnu/libglib-2.0.so.0
#16 0x00007fad5075230a in g_main_loop_run () from /lib/x86_64-linux-gnu/libglib-2.0.so.0
#17 0x00007fad4c998e25 in gtk_main () from /usr/lib/x86_64-linux-gnu/libgtk-3.so.0
#18 0x00007fad50ad90e5 in pidgin_start (argc=1, argv=0x7ffeb0cdada8) at libpidgin.c:850
#19 0x00007fad50147ec5 in __libc_start_main (main=0x556d94140960 <main>, argc=1, argv=0x7ffeb0cdada8, init=<optimized out>, fini=<optimized out>, rtld_fini=<optimized out>, stack_end=0x7ffeb0cdad98) at libc-start.c:287
#20 0x0000556d941409de in _start ()

(gdb) cont
Continuing.
^C
Program received signal SIGINT, Interrupt.
0x00007fad4e1ee713 in webkit_dom_object_get_type () from /usr/lib/x86_64-linux-gnu/libwebkitgtk-3.0.so.0
(gdb) where
#0  0x00007fad4e1ee713 in webkit_dom_object_get_type () from /usr/lib/x86_64-linux-gnu/libwebkitgtk-3.0.so.0
#1  0x00007fad4e27ffe6 in ?? () from /usr/lib/x86_64-linux-gnu/libwebkitgtk-3.0.so.0
#2  0x00007fad4e2825a8 in webkit_dom_node_get_parent_node () from /usr/lib/x86_64-linux-gnu/libwebkitgtk-3.0.so.0
#3  0x00007fad50acceff in do_popup_menu (webview=webview@entry=0x556d963e1290, button=3, time=424808907, context=6, node=0x556d961fa360,

```
uri=0x556d96208400 "https://www.facebook.com/ProgrammersCreateLife/photos/a.241809332534619.55240.241806149201604/980463552002523/?type=3&theater") at gtkwebview.c:768
```
#4  0x00007fad50acdca5 in webview_button_pressed (webview=0x556d963e1290, event=0x556d97113ba0) at gtkwebview.c:988
#5  0x00007fad4c999c4c in ?? () from /usr/lib/x86_64-linux-gnu/libgtk-3.so.0
#6  0x00007fad4b9753b8 in g_closure_invoke () from /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0
#7  0x00007fad4b986d3d in ?? () from /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0
#8  0x00007fad4b98e6f9 in g_signal_emit_valist () from /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0
#9  0x00007fad4b98ece2 in g_signal_emit () from /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0
#10 0x00007fad4cac2974 in ?? () from /usr/lib/x86_64-linux-gnu/libgtk-3.so.0
#11 0x00007fad4c99808c in ?? () from /usr/lib/x86_64-linux-gnu/libgtk-3.so.0
#12 0x00007fad4c99991a in gtk_main_do_event () from /usr/lib/x86_64-linux-gnu/libgtk-3.so.0
#13 0x00007fad4c59dc22 in ?? () from /usr/lib/x86_64-linux-gnu/libgdk-3.so.0
#14 0x00007fad50751e04 in g_main_context_dispatch () from /lib/x86_64-linux-gnu/libglib-2.0.so.0
#15 0x00007fad50752048 in ?? () from /lib/x86_64-linux-gnu/libglib-2.0.so.0
#16 0x00007fad5075230a in g_main_loop_run () from /lib/x86_64-linux-gnu/libglib-2.0.so.0
#17 0x00007fad4c998e25 in gtk_main () from /usr/lib/x86_64-linux-gnu/libgtk-3.so.0
#18 0x00007fad50ad90e5 in pidgin_start (argc=1, argv=0x7ffeb0cdada8) at libpidgin.c:850
#19 0x00007fad50147ec5 in __libc_start_main (main=0x556d94140960 <main>, argc=1, argv=0x7ffeb0cdada8, init=<optimized out>, fini=<optimized out>, rtld_fini=<optimized out>, stack_end=0x7ffeb0cdad98) at libc-start.c:287
#20 0x0000556d941409de in _start ()

(gdb)

**Comment by xhaakon** [comment]
Reproducible; happens whenever one right-clicks a link in conversation window.

**Comment by gy-lehel** [comment]
This issue is solved in release 1:3.0.0~collab-0.20602+20160
